client/my-sites/domains/domain-management/change-site-address/index.jsx

```diff
@@ -41,7 +41,7 @@ class ChangeSiteAddress extends Component {
 
 		return (
 			<Main className="change-site-address">
-				<Header onClick={ this.goBack } selectedDomainName={ domain.name }>
+				<Header onClick={ this.goBack } selectedDomainName={ get( domain, 'name', '' ) }>
```

Comment:
It looks like we're trying not to use lodash going forward: p4TIVU-9Bf-p2
Can we use a native function or operator to get the same result?

Reply:
Yeah, I think we're trying to remove lodash as a dependency where possible. How about:

```suggestion
				<Header onClick={ this.goBack } selectedDomainName={ domain?.name }>
```

Or if it has to be an empty string:

```suggestion
				<Header onClick={ this.goBack } selectedDomainName={ domain?.name ?? '' }>
```

Reply:
I really like the second one didn't know it was possible to do something like this with the `??` @pablinos! Is this the same as `domain ? domain.name  : ""`

Reply:
It looks like we lost that change, maybe because of the rebase?

Reply:
it seems to be the case, probably my git client is just playing some games with me... will add them again.